### ISSUE TITLE
start writing JOSS paper

### ISSUE THREAD

**Issue by rbturnbull** [jjmccollum/teiphy#30]
we can copy template paper.md from an example in the JOSS docs. We can also set up a bibtex file.

**Comment by jjmccollum** [comment]
@rbturnbull I've filled out the Statement of Need section. I may have been a bit too wordy, so feel free to suggest any edits or trim whatever you think is unnecessary. If the Design section needs more content, just let me know. I can talk about the UBS Ephesians use case, but it looks like you had an unfinished comment asking about another text that could be used as an example, so if you have something else in mind, feel free to mention it here or add it to the paper!

I also filled out the acknowledgment of Stephen's help and added a subsection acknowledging that the work was funded by an Australian Government RTP Scholarship. I don't know if I need to clarify that that just covers me or if there's anything else that you need to add to that, but feel free to suggest or make any changes on that, too!

**Comment by jjmccollum** [comment]
@rbturnbull I think the paper is ready. I removed the placeholder text ("Is there an example from") that you had at the end of the Use Case section; if there's something that you had in mind, feel free to add it to the discussion in that section. If there's anything else you think we should add, just let me know. Thanks!

**Comment by rbturnbull** [comment]
great work @jjmccollum - i'll have a read of the paper in the next few days and make some suggestions.

**Comment by jjmccollum** [comment]
@rbturnbull Per your suggestion in a recent pull request, I ran the output of

```
teiphy -t"reconstructed" -t"defective" -t"orthographic" -m"lac" -m"overlap" -s"*" -s"T" --fill-correctors --states-present example\ubs_ephesians.xml ubs_ephesians.nxs
```

through a heuristic tree search in PAUP*. I set it to save 10,000 trees and let it run for the day. Running on my laptop, the search terminated after about 5 hours and 45 minutes, and the best-found trees had a score of 376. I produced the 50% majority consensus tree and Adams consensus tree for the results and saved them (with different layouts) to PDFs. Here are the majority consensus trees:

[ubs-ephesians-majority-consensus.pdf](https://github.com/jjmccollum/teiphy/files/9546449/ubs-ephesians-majority-consensus.pdf)
[ubs-ephesians-majority-consensus-circle.pdf](https://github.com/jjmccollum/teiphy/files/9546450/ubs-ephesians-majority-consensus-circle.pdf)
[ubs-ephesians-majority-consensus-unrooted.pdf](https://github.com/jjmccollum/teiphy/files/9546451/ubs-ephesians-majority-consensus-unrooted.pdf)

And here are the Adams consensus trees:

[ubs-ephesians-adams-consensus.pdf](https://github.com/jjmccollum/teiphy/files/9546459/ubs-ephesians-adams-consensus.pdf)
[ubs-ephesians-adams-consensus-circle.pdf](https://github.com/jjmccollum/teiphy/files/9546460/ubs-ephesians-adams-consensus-circle.pdf)
[ubs-ephesians-adams-consensus-unrooted.pdf](https://github.com/jjmccollum/teiphy/files/9546461/ubs-ephesians-adams-consensus-unrooted.pdf)

**Comment by jjmccollum** [comment]
@rbturnbull I have my account set up with _JOSS_ and can submit the paper today if you think it is ready for submission; just let me know!

**Comment by rbturnbull** [comment]
Hi @jjmccollum - i think everything is ready for submission. Terrific work on this. you can submit when you like :tada:

**Comment by jjmccollum** [comment]
@rbturnbull All right the paper has been submitted, and I've received the automated confirmation of submission! Thanks so much for all your help on this; I couldn't have made this project nearly as well-maintained, documented, and accessible as it is without you.

**Comment by rbturnbull** [comment]
congratulations @jjmccollum - terrific work on this
